x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.test.ts

```diff
@@ -134,6 +134,7 @@ const baseRuleParams: Pick<RuleParams, 'count' | 'timeSize' | 'timeUnit'> = {
 
 const TIMESTAMP_FIELD = '@timestamp';
 const FILEBEAT_INDEX = 'filebeat-*';
+const TIMESTAMP = Date.now();
```

Comment:
To prevent accidentally introducing a time dependency into the unit test, would there be any downside in using a fixed value here?

Reply:
Nope... we can easily set that to a fixed value.